doc/index.rst

```diff
@@ -13,22 +13,42 @@ metrics.
 This package contains efficient Python implementations of several popular
 metric learning algorithms.
 
+Supervised Algorithms
+---------------------
+Supervised metric learning algorithms take as inputs points `X` and targets
+`y`, and learn to make points from the same class close to each other, and
```

Comment:
target class labels

Reply:
maybe target labels only, to include `MLKR` ?

Reply:
:+1:

Reply:
Just realized the rest of the sentence ("[...]from the same class[...]") didn't include `MLKR` so I'll update this too